tests/stress.py

```diff
@@ -10,6 +10,11 @@
 @pytest.fixture
 def configure():
     config_testsuite()
+    config_merge({'extensions': 
+            { 'haas.ext.obm.ipmi': '',
+              },
+                    })
+
```

Comment:
Indeed. @SahilTikale, since the pep8 patch hasn't landed yet, would you run one of the pep8 linters on this branch for us?